### PR TITLE
[reorg] Move ICSL script out of BERT folder

### PR DESCRIPTION
**Pull request by eric-haibin-lin**
## Description ##
A step further to make the bert folder thinner. Also add test for the script. 

### Essentials ###
- [x] PR's title starts with a category (e.g. [BUGFIX], [MODEL], [TUTORIAL], [FEATURE], [DOC], etc)
- [x] Changes are complete (i.e. I finished coding on this PR)
- [x] All changes have test coverage
- [x] Code is well-documented

### Changes ###
- [ ] Feature1, tests, (and when applicable, API doc)
- [ ] Feature2, tests, (and when applicable, API doc)

## Comments ##
- If this change is a backward incompatible change, why must this change be made.
- Interesting edge cases to note here

### CODE REVIEW

**Comment by mli** [comment]
Job PR-1131/1 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-1131/1/index.html

**Comment by mli** [comment]
Job PR-1131/3 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-1131/3/index.html

**Comment by mli** [comment]
Job PR-1131/2 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-1131/2/index.html

File: scripts/intent_cls_slot_labeling/index.rst [hunk 1]
```diff
@@ -0,0 +1,106 @@
+Intent Classification and Slot Labeling
+---------------------------------------
+
+:download:`Download scripts </model_zoo/intent_cls_slot_labeling.zip>`
+
+Reference: Devlin, Jacob, et al. "`Bert: Pre-training of deep bidirectional transformers for language understanding. <https://arxiv.org/abs/1810.04805>`_" arXiv preprint arXiv:1810.04805 (2018).
```
Comment (sxjscience) [inline scripts/intent_cls_slot_labeling/index.rst]: For the references, we use 

```
- Chen, Qian, et al. "`BERT for Joint Intent Classification and Slot Filling. <https://arxiv.org/pdf/1902.10909.pdf>`"  arXiv preprint arXiv:1902.10909 (2019).
- Devlin, Jacob, et al. "`Bert: Pre-training of deep bidirectional transformers for language understanding. <https://arxiv.org/abs/1810.04805>`_"  arXiv preprint arXiv:1810.04805 (2018).
```

**Comment by mli** [comment]
Job PR-1131/4 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-1131/4/index.html

**Comment by mli** [comment]
Job PR-1131/5 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-1131/5/index.html

**Comment by mli** [comment]
Job PR-1131/6 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-1131/6/index.html
